public/js/components/card-input.jsx

```diff
@@ -98,14 +98,23 @@ export default class CardInput extends Component {
                       errorModifier={this.props.errorModifier} /> : null }
         { showCardIcon ?
           <PayMethodIcon payMethodType={this.props.cardType} /> : null }
-        <MaskedInput
-          {...this.props.attrs}
-          id={this.props.id}
-          className={this.props.id + '-input'}
-          onChange={this.props.onChangeHandler}
-          pattern={pattern}
-          placeholder={placeholder}
-        />
```

Comment:
oh nevermind, I see that one is a regular input and another is MaskedInput. I guess copying the attributes is fine.